### PR TITLE
class library: make sure color is not converted to array

### PR DESCRIPTION
**Pull request by telephon**
## Purpose and Motivation

The example:
```supercollider
(
a = { (0..30).scramble }.dup(2).plot;
a.setProperties(
    \fontColor, Color.red,
    \plotColor, Color.red,
    \backgroundColor, Color.gray,
    \gridColorX, Color.white,
    \gridColorY, Color.black,

    \labelX, "Humidity"
);
a.refresh;
);
```
would not have a red `plotColor` because it was internally converted to an array (`Color`unfortunately responds to `asArray`with an array of values).

This fixes it.

## Types of changes

- Bug fix

## To-do list

- [x] Code is tested
- [x] This PR is ready for review

### CODE REVIEW

**Comment by mtmccrea** [comment]
Thanks @telephon!

`Plotter:plotColor_` avoids this by:
https://github.com/supercollider/supercollider/blob/695160920998acc1a312d7e323c5f33255a0ce4d/SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc#L919-L920

Should we stick with this approach for consistency?

**Comment by telephon** [comment]
>Should we stick with this approach for consistency?

ok, I've added it here, too.

**Comment by telephon** [comment]
Ah I see, that is better.

File: SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc [hunk 1]
```diff
@@ -86,7 +86,7 @@ Plot {
 		})
 	}
 	plotColor_ { |c|
-		if(c.isArray.not) { c = c.bubble };
+		colors = colors.as(Array);
```
Comment (dyfer) [inline SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc:89]: should't this be `c = c.as(Array)`?

Comment (mtmccrea) [reply SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc:89]: The whole method can just be:
```
plotColor_ { |c|
		plotColor = c.as(Array);
}

Comment (telephon) [reply SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc:89]: yes, of course – thank you.

Comment (mtmccrea) [reply SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc]: Thanks! And sorry, I should have been more clear about the differences in `Plot:plotColor` and `Plotter:plotColor`.

**Review by mtmccrea — APPROVED** [review]

**Comment by dyfer** [comment]
Thanks!
For the future - when we have 5 commits to change one line of code, feel free to squash on merge :)

**Comment by telephon** [comment]
>EDIT: was this squashed on merge? I realized I'm not sure where to check this

Yes, sure. It would have been rather ridiculous otherwise ;-)

This is how I would check:

https://github.com/supercollider/supercollider/commits/develop
https://github.com/supercollider/supercollider/commit/b10674982af5acfebc7e07127f60db9f39758156

**Comment by dyfer** [comment]
Great, thanks!

**Pull request merged**